session_items.py

```diff
@@ -1,77 +1,65 @@
-from flask import session
+import requests
```

Comment:
You're no longer using the Flask session cookie state in this file. Should it be renamed?